### ISSUE TITLE
Cannot see "mods" in options

### ISSUE THREAD

**Issue by QqiqiQ** [SubnauticaModding/QModManager#273]
**Describe the problem**

I used the EXE file to install Qmod correctly, but I couldn't find the mods option and didn't mod didn't run
**Screenshots**

![屏幕截图(109)](https://user-images.githubusercontent.com/85970621/162613605-f5c9ce3e-e86d-4d98-a7de-163b9e2184c6.png)
![屏幕截图(110)](https://user-images.githubusercontent.com/85970621/162613609-02194f4d-0e9a-4071-ad81-08ae46884a5d.png)

**Additional info**

**Comment by desperationfighter** [comment]
Before i start let me recommen you to visit the Modding Discord [discord.gg/upwuwwq](https://discord.gg/UpWuWwq)  helping there is much easier.

However back to the Problem:
You have preloader Logfiles.
That means something broke on load.
To know what happens on your Side we need to check the log first.
So please provide/upload the latest "preloader_xyz" file in your case its "preloader_20220410_175845_320.log".

Also please upload your "qmodmanager_log-Subnautica.txt".

**Comment by QqiqiQ** [comment]
Here it is
[preloader_20220410_175845_320.log](https://github.com/SubnauticaModding/QModManager/files/8458815/preloader_20220410_175845_320.log)
[qmodmanager_log-Subnautica.txt](https://github.com/SubnauticaModding/QModManager/files/8458816/qmodmanager_log-Subnautica.txt)

**Comment by desperationfighter** [comment]
mhh i am a bit confused on some things you may want to clear up.
I see you named the Install Folder "_Linux" so are you running the Game on Linux ?
Also do you have the Game from Steam or from somewhere else ?

**Comment by QqiqiQ** [comment]
no,I run it on win10, it just the name
yes,I also download some other games from steam and other places

**Comment by desperationfighter** [comment]
OK Windows 10, weired why you call it Linux then.
However you didn't answered the second question. I was not asking you where you get your games in general. I asked if you get Subnautica From Steam.
But i suspect you did not. If this is true i need to deny my help on Pirated Copies.

**Comment by QqiqiQ** [comment]
oh sorry,I just find the game is for linux,it is in an installation file,I will try to download a new one
I download this game from my friend's computer ,and I don't know how he download it....

**Comment by QqiqiQ** [comment]
thank you

**Comment by celvro** [comment]
Also having the same issue, in my case I downloaded from Steam. It's weird because I was testing a mod I made last night and this morning the mods menu is gone. I don't know what I did, but it doesn't show up even if I verify game files in steam and then completely reinstall QMods, no preloaders or even a debug file from qmod after starting the game.

I'm going to completely re-install subnautica and try with just qmods
